src/util/string.ts

```diff
@@ -1,6 +1,19 @@
+import { isHex } from '../base/decoder/types';
+
 export function stripHexPrefix(data: string): string {
```

Comment:
can also use this library

https://www.npmjs.com/package/ethjs-util

Reply:
also useful: https://www.npmjs.com/package/ethereumjs-util

Reply:
thanks @ryan-bitgo 
the version of `ethereumjs-util` we are using (6.2.0) doesn't provide the methods: `stripHexPrefix, isHexPrefixed, isHexString, etc` 
Do you think it is worth upgrading it to 7.1.0 and running a regression?

Reply:
up to you. not a blocker.